src/popup/components/TransactionTokenRows.vue

```diff
@@ -82,11 +82,19 @@ export default defineComponent({
 
     const getTokenName = (token: ITokenResolved) => token?.isAe ? AE_SYMBOL : token.symbol;
 
+    const amountFormatted = ({ token, isRounded }: {token: ITokenResolved; isRounded: boolean}) => {
+      // if amount is in scientific notation it is not rounded
+      if (token.amount && (!isRounded || token.amount?.toString().indexOf('e') !== -1)) {
```

Comment:
Why this condition is needed? What was the case?

Reply:
BTC amount was shown in scientific notation